back/sssscs/src/main/java/com/ib/config/ControllerAdvisor.java

```diff
@@ -0,0 +1,21 @@
+package com.ib.config;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import com.ib.user.exception.EmailTakenException;
+
+import jakarta.servlet.http.HttpServletRequest;
+
+@ControllerAdvice
+@EnableWebMvc
+public class ControllerAdvisor extends ResponseEntityExceptionHandler {
```

Comment:
Remove @EnableWebMvc. Might cause issues later

Reply:
Fixed in [526576c](https://github.com/magley/sssscs/pull/1/commits/526576cf20344a23c92be7c7ecad2359058da605).